beacons/pom.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.gluonhq.samples</groupId>
+    <artifactId>beacons</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>beacons</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>11</maven.compiler.release>
+        <javafx.version>14.0.1</javafx.version>
+        <charm.version>6.0.4-SNAPSHOT</charm.version>
+        <attach.version>4.0.7</attach.version>
+        <javafx-maven-plugin-version>0.0.4</javafx-maven-plugin-version>
+        <client-maven-plugin-version>0.1.22-SNAPSHOT</client-maven-plugin-version>
```

Comment:
Why do we still need the snapshot version to run this sample?

Reply:
There are important changes merged after the last release.

Reply:
This can be updated now.

Reply:
Ok, I see this has been updated. Marking the conversation as "Resolved".